v5/pivotal/stories.go

```diff
@@ -155,6 +169,18 @@ type BlockerRequest struct {
 	Resolved    *bool  `json:"resolved,omitempty"`
 }
 
+// Review is a review on a PT story
+type Review struct {
+	ID           int          `json:"id,omitempty"`
+	StoryID      int          `json:"story_id,omitempty"`
+	ReviewTypeID int          `json:"review_type_id,omitempty"`
+	ReviewerID   int          `json:"reviewer_id,omitempty"`
+	Status       ReviewStatus `json:"status,omitempty"`
+	CreatedAt    *time.Time   `json:"created_at,omitempty"`
+	UpdatedAt    *time.Time   `json:"updated_at,omitempty"`
+	Kind         string       `json:"kind,omitempty"`
+}
```

Comment:
I am wondering whether we shouldn't use pointers somewhere, like `ReviewerID`, is that always set?